__tests__/app.test.js

```diff
@@ -113,6 +113,54 @@ describe("/api/articles", () => {
           expect(articles).toBeSortedBy("created_at", { descending: true });
         });
     });
+    test("GET 200: takes a sort_by query that sorts by any valid column", () => {
+      return request(app)
+        .get("/api/articles?sort_by=title")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles.length).toBe(13);
+          expect(articles).toBeSortedBy("title", { descending: true });
+        });
+    });
+    test("GET 200: takes a order query, asc sorts by ascending", () => {
+      return request(app)
+        .get("/api/articles?order=desc")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles.length).toBe(13);
+          expect(articles).toBeSortedBy("created_at", { descending: true });
+        });
+    });
```

Comment:
Nice, but this says asc in the description but you are testing for descending in the actual assertion. I believe the default behaviour should be already sorted by descending so this assertion should be updated to actually be sorted by ascending with the correct changes also made to the endpoint request as well on line 127.